packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -34,9 +34,13 @@ import { WorkspaceDeleteHandler } from './workspace-delete-handler';
 const validFilename: (arg: string) => boolean = require('valid-filename');
 
 export namespace WorkspaceCommands {
-    export const OPEN: Command = {
-        id: 'workspace:open',
-        label: 'Open...'
+    export const OPEN_FILE: Command = {
+        id: 'workspace:openFile',
+        label: 'Open File...'
```

Comment:
Why not `Open...` as before? It is the same in VS Code.

Reply:
Yes, looks like that:
<img width="359" alt="screen shot 2018-08-14 at 23 28 57" src="https://user-images.githubusercontent.com/3082655/44119589-f4f54c66-a019-11e8-9800-c5f2051e2e65.png">

What is `Open Folder...`? What is tha label of command in the pallete, also `File: Open File...` or `File: Open...`?

Reply:
in the ubuntu version of vsCode, 
- `Open File` opens a single file, e.g., settings.json.
- `Open Folder` opens a directory from the filesystem.
- `Open Workspace` recreates a workspace, which contains "0 ~ many" projects, from a `code-workspace` file.

My plan is having `Open File...` and `Open Workspace...`, where `Open File...` opens a single file, and `Open Workspace...` either opens a folder, or recreates a multi-root workspace from a config file (in near future).

as always, I am open to suggestions :)

Reply:
@elaihau  it is bogus, `File Open...` already reserved for the search:
<img width="663" alt="screen shot 2018-08-14 at 23 39 19" src="https://user-images.githubusercontent.com/3082655/44120021-4b7cebf6-a01b-11e8-9b70-471ae9b7713c.png">

We should either revert to `Open...` or disambiguate it somehow else. I would vote for reverting since it will also avoid a breaking change by renaming a constant from `WorkspaceCommands.OPEN` to `WorkspaceCommands.OPEN_FILE`.  I am also not sure why `Open` cannot open folders as workspaces. I thought the issues were only to limit `Open Worskpace...` to folders, not changing semantic of the open command.

Reply:
Thank you for pointing out the problem this change has. Really appreciated !

The potential issue of having only one `Open...` is that, what do we do if the user hits the `Open...` button, and double clicks a workspace config file, like `my_workspace.code-workspace`? Would we open that file in the editor, or recreate the workspace from it?

Reply:
> The potential issue of having only one Open... is that, what do we do if the user hits the Open... button, and double clicks a workspace config file, like my_workspace.code-workspace? Would we open that file in the editor, or recreate the workspace from it?

How is it different to `Open File...`? We just open a file in both cases. If a user wants to open a workspace he should use `Open Workspace...`.

Reply:
hmm... just want to make sure I don't misunderstand: are you suggesting having 2 menu items: `Open...` (opens file in editor) and `Open Workspace...` (opens folder or recreates workspace) ?

Thanks !

Reply:
I suggest leaving `Open...` as it used to work. Keep `Open Workspace...` as it works in this PR.

Reply:
Got it. I will revert this change partially and start one more PR.